index.js

```diff
@@ -80,14 +80,12 @@ export default class TinySDF {
         gridOuter.fill(INF, 0, len);
         gridInner.fill(0, 0, len);
 
-        const offset = (width - glyphWidth) >> 1;
```

Comment:
:+1: Yeah, the offset was left over from when width/glyphWidth could be different by a variable amount.